src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -8,45 +8,145 @@
+            .open(MercurySubpages.MAIN_PAGE)
+            .getTopBar()
+            .openSearch();
+
+    Assertion.assertTrue(search.isSearchInputFieldVisible());
+    Assertion.assertTrue(search.isClearSearchButtonVisible());
+    Assertion.assertTrue(search.isInputFieldSearchIconVisible());
+  }
+
+  @Test(groups = "mercury_search_userIsRedirectedToSearchResultsPage")
+  public void mercury_search_userIsRedirectedToSearchResultsPage() {
+    SearchResultsPage searchResults =
+        new ArticlePage()
+            .open(MercurySubpages.MAIN_PAGE)
+            .getTopBar()
+            .openSearch()
+            .typeInSearch(this.SEARCH_PHRASE)
+            .clickEnterAndNavigateToSearchResults();
+
+    Assertion.assertTrue(searchResults.isSearchResultsPageOpen());
+  }
+
+  @Test(groups = "mercury_search_searchResultsPageHasNoSearchIconInTopBar")
+  public void mercury_search_searchResultsPageHasNoSearchIconInTopBar() {
+    SearchResultsPage resultsPage =
+        new SearchResultsPage()
+            .openForQuery(SEARCH_PHRASE);
+
+    Assertion.assertFalse(resultsPage.getTopBar().isSearchIconClickable());
+  }
+
+  @Test(groups = "mercury_search_searchInputDoesNotCoverNavigation")
+  public void mercury_search_searchInputDoesNotCoverNavigation() {
+    SearchResultsPage resultsPage =
+        new SearchResultsPage()
+            .openForQuery(SEARCH_PHRASE);
+
+    resultsPage
+        .getTopBar()
+        .openNavigation();
+
+    Assertion.assertFalse(resultsPage.getSearch().isSearchInputFieldEditable());
+  }
+
+  @Test(groups = "mercury_search_searchNoResultsPageDisplayed")
+  public void mercury_search_searchNoResultsPageDisplayed() {
+    SearchResultsPage searchResults =
+        new SearchResultsPage()
+            .openForQuery(SEARCH_PHRASE_NO_RESULTS);
+
+    Assertion.assertTrue(searchResults.isNoResultsPagePresent());
+  }
+
+  @Test(groups = "mercury_search_noSuggestionsOnSearchResultsPage")
+  public void mercury_search_noSuggestionsOnSearchResultsPage() {
+    SearchResultsPage searchResults =
+        new SearchResultsPage()
+            .openForQuery(SEARCH_PHRASE_NO_RESULTS);
+
+    Assertion.assertFalse(searchResults.getSearch().areSearchSuggestionsDisplayed());
+  }
+
+  @Test(groups = "mercury_search_focusOnTryAnotherSearchWhenNoResults")
+  public void mercury_search_focusOnTryAnotherSearchWhenNoResults() {
+    SearchResultsPage searchResults =
+        new SearchResultsPage()
+            .openForQuery(SEARCH_PHRASE_NO_RESULTS)
+            .clickTryAnotherSearch();
+
+    Assertion.assertTrue(searchResults.getSearch().isInputFieldFocused());
+    Assertion.assertTrue(searchResults.getSearch().getSearchPhrase().isEmpty());
+  }
 
-    Search search = topBar.openSearch();
-    search.typeInSearch(searchingPhrase);
-    search.clickClearSearchButton();
+  @Test(groups = "mercury_search_redirectToNewResultsPageFromNoResults")
+  public void mercury_search_redirectToNewResultsPageFromNoResults() {
```

Comment:
is it possible to check also if the underline is blue and small search icon on the left has proper color?